src/Struct/FindologicProduct.php

```diff
@@ -356,11 +356,11 @@ public function getKeywords(): array
 
     protected function setKeywords(): void
     {
-        $tags = $this->product->getTags();
-        if ($tags !== null && $tags->count() > 0) {
-            foreach ($tags as $tag) {
-                if (!Utils::isEmpty($tag->getName())) {
-                    $this->keywords[] = new Keyword($tag->getName());
+        $keywords = $this->product->getSearchKeywords();
```

Comment:
@D3xter7 We need to remove the vendor name and product number from here as well.